### DISCUSSION TITLE
Why was JAXLIB 0.1.75 removed from pypi?

### DISCUSSION THREAD

**Discussion by kevExternal** [jax-ml/jax#18045] [Q&A] [answered]
Hi everyone, I just noticed recently while trying to build my project that Jaxlib 0.1.75 was remove from PYPI https://pypi.org/project/jaxlib/#history
Can someone provide insight why it was removed? Upgrading to 0.3.x breaks lots of code. Any insight on reason for package removal would be appreciated. I still see it under https://storage.googleapis.com/jax-releases/jax_releases.html

**Comment by jakevdp** [comment, marked as answer]
pypi has limited storage capacity, so we periodically remove the wheels for old releases in order to make room for new ones. You can install older wheels by pointing `pip` at the storage bucket you mention, e.g.
```
pip install jaxlib==0.1.75 -f https://storage.googleapis.com/jax-releases/jax_releases.html
```
